gradle-jdks-excavator-configurations/src/main/java/com/palantir/gradle/jdks/GradleJdksConfigurator.java

```diff
@@ -68,28 +61,18 @@ private static void writeJdkInstallationConfiguration(
             String jdkVersion,
             Os os,
             Arch arch,
-            Optional<String> palantirCert,
             Path targetDir) {
-        JdkSpec.Builder jdkSpecBuilder = JdkSpec.builder()
-                .distributionName(jdkDistributionName)
-                .release(JdkRelease.builder()
-                        .arch(arch)
-                        .os(os)
-                        .version(jdkVersion)
-                        .build());
-        palantirCert.ifPresentOrElse(
-                cert -> jdkSpecBuilder.caCerts(CaCerts.from(Map.of(PALANTIR_ALIAS_CERT, cert))),
-                () -> jdkSpecBuilder.caCerts(CaCerts.from(Map.of())));
-        JdkSpec jdkSpec = jdkSpecBuilder.build();
+        JdkRelease jdkRelease =
+                JdkRelease.builder().arch(arch).os(os).version(jdkVersion).build();
 
         Path jdksDir = targetDir.resolve("jdks");
         Path jdkOsArchDir = jdksDir.resolve(javaVersion).resolve(os.uiName()).resolve(arch.uiName());
         try {
             Files.createDirectories(jdkOsArchDir);
             GradleJdksConfigsUtils.writeConfigurationFile(
-                    jdkOsArchDir.resolve("download-url"), resolveDownloadUrl(baseUrl, jdkSpec));
+                    jdkOsArchDir.resolve("download-url"), resolveDownloadUrl(baseUrl, jdkDistributionName, jdkRelease));
             GradleJdksConfigsUtils.writeConfigurationFile(
-                    jdkOsArchDir.resolve("local-path"), resolveLocalPath(jdkSpec));
+                    jdkOsArchDir.resolve("local-path"), resolveLocalPath(jdkDistributionName, jdkRelease));
```

Comment:
So we are keeping the local path? I think this may make sense tbh.

Reply:
Yes, I also think this might be better. It would have ended up a little bit too complicated to extract the local-path in bash given that the downloadUrls might have had multiple dots also in the name of the file and the name of the extensions (tar.bz2 etc. )